### PR TITLE
Bump libreoffice, please

### PR DESCRIPTION
**Pull request by holgersson32644**
- app-office/libreoffice: Update live ebuild
- app-office/libreoffice-l10n: Bump to 7.2.0.4
- app-office/libreoffice: Bump to 7.2.9999 and 7.2.0.4

### CODE REVIEW

**Comment by holgersson32644** [comment]
Had to drop the patches from libreoffice-9999 (which took about 8h to fetch everything from a slow upstream git repo).

**Comment by jospezial** [comment]
This has some problems.
```
		local base_uri branch mypv
		base_uri="https://anongit.freedesktop.org/git"
		branch="master"
		mypv=${MY_PV/.9999}
		[[ ${mypv} != ${MY_PV} ]] && branch="${PN}-${mypv/./-}"
		git-r3_fetch "${base_uri}/${PN}/core" "refs/heads/${branch}"
		git-r3_checkout "${base_uri}/${PN}/core"
		LOCOREGIT_VERSION=${EGIT_VERSION}
		git-r3_fetch "${base_uri}/${PN}/help" "refs/heads/master"
		git-r3_checkout "${base_uri}/${PN}/help" "helpcontent2" # doesn't match on help
```
The upper lines fetch all other repos because they are git submodules (for example 6GByte translations that are not used by the live ebuild). help is done twice
If I look at https://www.libreoffice.org/download/pre-releases/ there is https://git.libreoffice.org/core as clone url.
I changed these in my local ebuild.
```
		local base_uri branch mypv
		base_uri="https://git.libreoffice.org"
		branch="master"
		EGIT_SUBMODULES=(helpcontent2)
		mypv=${MY_PV/.9999}
		[[ ${mypv} != ${MY_PV} ]] && branch="${PN}-${mypv/./-}"
		git-r3_fetch "${base_uri}/core" "refs/heads/${branch}"
		git-r3_checkout "${base_uri}/core"
		LOCOREGIT_VERSION=${EGIT_VERSION}

#		git-r3_fetch "${base_uri}/help" "refs/heads/master"
#		git-r3_checkout "${base_uri}/help" "helpcontent2" # doesn't match on help
```

**Comment by jospezial** [comment]
Why do we not use the standard gentoo combined release/live ebuild design for the sources?

File: app-office/libreoffice/libreoffice-9999.ebuild [hunk 1]
```diff
@@ -463,7 +462,6 @@ src_configure() {
 	# --without-system-sane: just sane.h header that is used for scan in writer,
 	#   not linked or anything else, worthless to depend on
 	# --disable-pdfium: not yet packaged
-	# --without-system-qrcodegen: has no real build system and LO is the only user
 	local myeconfargs=(
 		--with-system-dicts
 		--with-system-epoxy
```
Comment (jospezial) [inline app-office/libreoffice/libreoffice-9999.ebuild:467]: drop --disable-neon. Use --with-webdav instead
https://bugs.gentoo.org/793773

File: app-office/libreoffice/libreoffice-9999.ebuild [hunk 2]
```diff
@@ -507,7 +505,6 @@ src_configure() {
 		--without-system-jfreereport
 		--without-system_apache_commons
```
Comment (jospezial) [inline app-office/libreoffice/libreoffice-9999.ebuild:506]: https://bugs.gentoo.org/793776
https://github.com/LibreOffice/core/commit/6de0b1710adfba82c96b75a5da6f52633a54c692
"Get rid of apache-commons-logging"

**Comment by jospezial** [comment]
https://github.com/LibreOffice/core/releases/tag/libreoffice-7.2.2.1

**Comment by jospezial** [comment]
https://bugs.gentoo.org/813399

**Comment by holgersson32644** [comment]
Hi, I didn't forget or oversaw your notes here, but I currently don't have enough time to address them, sorry about that.

**Comment by jospezial** [comment]
Needed to depclean :12 . Compile tested with my libreoffice-9999.ebuild
```
	clang? (
		|| (
			(	sys-devel/clang:13
				sys-devel/llvm:13
				=sys-devel/lld-13*	)
```

**Pull request closed without merging**